zqd/search/search.go

```diff
@@ -25,6 +25,11 @@ const DefaultMTU = 100
 
 const StatsInterval = time.Millisecond * 500
 
+const (
+	MimeTypeNDJSON = "application/ndjson"
```

Comment:
https://github.com/ndjson/ndjson-spec#33-mediatype-and-file-extensions says
```suggestion
	MimeTypeNDJSON = "application/x-ndjson"
```

Reply:
Actually @nwt  the `x-` convention has been deprecated. Elastic also just does `application/ndjson`. We've had this discussion on another pr.

Reply:
@mattnibs: I don't see application/ndjson in https://github.com/elastic/elasticsearch, but I do see application/x-ndjson.

Reply:
Looks like they were going to do it but decided it would be too big of hassle: https://github.com/elastic/elasticsearch/issues/25718

Reply:
Whatever we do we should remain consistent. Because in other places we are consistently doing `application/ndjson` we should go with that, then consider if we want to switch to `x-ndjson` in another pr.

Reply:
I looked in the Brim & zq repo, and didn't see any other references for ndjson, where else do we use `application/ndjson`?